src/services/TechRecordsService.ts

```diff
@@ -365,6 +366,39 @@ class TechRecordsService {
     }
   }
 
+  public async managePlatesGeneration(techRecord: ITechRecord) {
+    let platesGenerated = false;
+    if (techRecord.plates) {
```

Comment:
Need to confirm. Does this mean that we need to have plates as undefined if don't want to generate plates?

Reply:
Plates are generated if there's an object in the plates array without plateSerialNumber and all other fields filled in. If you don't want to generate plates you simply send back the same plates array that you received on GET